hack/update-codegen.sh

```diff
@@ -20,52 +20,160 @@ trap cleanup EXIT
 
 go install ./${CODEGEN_PKG}/cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
 
+go build k8s.io/code-generator/cmd/applyconfiguration-gen
```

Comment:
Build from vendor else this will pull down fresh deps every time we run this in CI

```suggestion
go build ./vendor/k8s.io/code-generator/cmd/applyconfiguration-gen
```

As per suggestion above, might be better to build into an output dir too

```suggestion
goos=$(go env GOOS)
goarch=$(go env GOARCH)

APPLYCONFIGURATION_GEN=./_output/bin/${goos}/${goarch}/applyconfiguration-gen
go build -o ${APPLYCONFIGURATION_GEN} ./vendor/k8s.io/code-generator/cmd/applyconfiguration-gen
```

I'm including the GOOS and GOARCH in there because I keep having to delete the binaries when I switch between generating in container vs locally on mac

Then just execute with `${APPLYCONFIGURATION_GEN}` below

Reply:
done